src/components/ContributorShowcase/ContributorShowcase.module.css

```
@@ -0,0 +1,41 @@
+@media (min-width: 300px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto;
+  }
+}
+
+
+@media (min-width: 768px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto auto;
+  }
+}
+
+@media (min-width: 992px) { 
+  .grid {
+    display: grid;
+    grid-template-columns: auto auto auto auto auto auto auto;
```

looks a little weird imo with 7 in a row, let's go with 5 for now and then change in the future if it gets too big.
```suggestion
    grid-template-columns: auto auto auto auto auto;
```